database/migrations/0001_initial.py

```diff
@@ -112,7 +112,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('prime_id', models.CharField(blank=True, max_length=9, verbose_name='PrIMe ID')),
-                ('formula', models.CharField(max_length=50)),
+                ('formula', models.CharField(max_length=50, unique=True)),
```

Comment:
Is this saying each Species has a unique formula? Not true.

Even "O" has multiple species (the singlet O atom and a triplet O atom, with different spin states, are represented as different species in some models).  By the time you get to [C5H10O](https://www.google.com/search?q=C5H10O) you'll see there are many possibilities.

But maybe I misunderstood.

Reply:
I was thinking that it would be something like this in the specific instance of O:
1 O Species: related to all information to species with exactly 1 oxygen atom
1 O Isomer: there is only 1 bonding configuration (maybe I'm wrong here because of O2? not sure)
2 O Structures: singlet & triplet

We should chat about this more during out next meeting to clear it up